uint8 FILL_LINEAR=0
uint8 FILL_CONCENTRIC=1
uint8 fill_type
float64 angle
float64 distance
float64 outer_offset
uint8 outline_count
uint8 outline_overlap_count
geometry_msgs/Polygon outline
geometry_msgs/Polygon[] holes
---
Path[] paths